src/librustdoc/doctest.rs

```diff
@@ -111,19 +113,16 @@ crate fn run(options: Options) -> Result<(), ErrorReported> {
     let res = interface::run_compiler(config, |compiler| {
         compiler.enter(|queries| {
             let _lower_to_hir = queries.lower_to_hir()?;
-
-            let crate_name = queries.crate_name()?.peek().to_string();
             let mut global_ctxt = queries.global_ctxt()?.take();
 
             let collector = global_ctxt.enter(|tcx| {
                 let krate = tcx.hir().krate();
                 let crate_attrs = tcx.hir().attrs(CRATE_HIR_ID);
 
-                let mut opts = scrape_test_config(crate_attrs);
-                opts.display_warnings |= options.display_warnings;
+                let opts = scrape_test_config(crate_attrs);
```

Comment:
Err. I accidentally removed this line.

Reply:
Whoops, my fault for missing that. Honestly, I thought it was part of your changes to name handling at first.

Reply:
No big deal, I missed it too 😆